fuel-vm/src/tests/encoding.rs

```diff
@@ -217,7 +217,6 @@ fn transaction() {
             vec![0xfa],
             vec![0xfb, 0xfc],
             Policies::new()
-                .with_gas_price(Word::MAX >> 1)
```

Comment:
Could you also use tip here instead of the removing it, please?=) It is related to the encoding, so it is okay to keep